src/Algolia.Search/Models/Settings/IndexSettings.cs

```diff
@@ -282,10 +282,32 @@ private List<string> Slaves
 
         // performance
 
+        private bool _numericAttributesForFilteringWasSet = false;
+
+        /// <summary>
+        /// List of numeric attributes that can be used as numerical filters.
```

Comment:
Please explain why this method is here and link to the documentation: 
https://www.newtonsoft.com/json/help/html/ConditionalProperties.htm